common/tools/dev-tool/src/commands/admin/migrate-package.ts

```diff
@@ -264,6 +269,31 @@ function fixSourceFiles(packageFolder: string): void {
   }
 }
 
+function fixNodeDeclaration(moduleSpecifier: string): string {
+  const nodeModules = [
+    "assert",
+    "crypto",
+    "events",
+    "fs",
+    "fs/promises",
+    "http",
+    "https",
+    "net",
+    "os",
+    "path",
+    "process",
```

Comment:
This is good! But it would prevent people from polyfilling for browsers some `process` API which  our AMQP dependency is using?

Reply:
We shouldn't be using anything `process` related in our own code.  For external modules they may need process, but we don't.